signalfx-java-examples/src/main/java/com/signalfx/example/ProtobufExample.java

```diff
@@ -0,0 +1,100 @@
+package com.signalfx.example;
```

Comment:
New files should have our copyright at the top.

/**
 * Copyright (C) 2016 SignalFx, Inc. All rights reserved.
 */